tests/meta/qc_metrics/conftest.py

```diff
@@ -0,0 +1,96 @@
+"""Fixtures for the collect qc metrics api."""
+
+from datetime import datetime
+from pathlib import Path
+
+import pytest
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.clients.arnold.api import ArnoldAPIClient
+from cg.clients.janus.api import JanusAPIClient
+from cg.meta.qc_metrics.collect_qc_metrics import CollectQCMetricsAPI
+from cg.models.cg_config import CGConfig
+from cg.store.store import Store
+from tests.store_helpers import StoreHelpers
+
+
+@pytest.fixture
+def janus_hk_tags() -> list[str]:
+    return ["qc-metrics", "janus", "multiqc"]
+
+
+@pytest.fixture
+def file_with_right_tags() -> str:
+    return Path("tests", "fixtures", "meta", "qc_metrics", "file_with_right_tags.json").__str__()
+
+
+@pytest.fixture
+def file_without_right_tags() -> str:
+    return Path("tests", "fixtures", "meta", "qc_metrics", "file_without_right_tags.json").__str__()
+
+
+@pytest.fixture(scope="function")
```

Comment:
I think the scope is function by default but maybe someone else can confirm that